experimental/traffic_router_golang/README.md

```diff
@@ -17,4 +17,51 @@
     under the License.
 -->
 
+# Traffic Router 
+
 This is a prototype of Traffic Router's HTTP side in Golang.
+
+# How to build
+
+To get this app running locally:
+
+- Clone this repo
+- Install Golang programming language ([instructions](https://golang.org/doc/install))
+
+
+# Configuration
+
+Sample configuration file(cfg.json) available in traffic_router_golang directory, please add coveragezone files to path specified in cfg.json
+   
+
+# Build
+
+Compile and generate binary:
+
+   - `cd traffic_router_goland`
+   - `go mod vendor`
+   - `go build` #This will generate binary file traffic_router_golang)
+
+# Unit Test
+    
+   - Run `go test ./...` from traffic_router_golang directory
+     
+     ```$ go test ./...
+		?       traffic_router_golang   [no test files]
+		?       traffic_router_golang/availableservers  [no test files]
+		?       traffic_router_golang/cgsrch    [no test files]
+		?       traffic_router_golang/config    [no test files]
+		?       traffic_router_golang/coveragezone      [no test files]
+		?       traffic_router_golang/crconfig  [no test files]
+		?       traffic_router_golang/crconfigdsservers [no test files]
+		?       traffic_router_golang/crconfigpoller    [no test files]
+		?       traffic_router_golang/crconfigregex     [no test files]
+		?       traffic_router_golang/crstates  [no test files]
+		?       traffic_router_golang/crstatespoller    [no test files]
+		?       traffic_router_golang/fetch     [no test files]
+		?       traffic_router_golang/httpsrvr  [no test files]
+		?       traffic_router_golang/ipmap     [no test files]
+		?       traffic_router_golang/nextcache [no test files]
+		ok      traffic_router_golang/quadtree  0.526s
+		?       traffic_router_golang/toutil    [no test files]
```

Comment:
I get different output when running `go test ./...`:

```
?   	github.com/apache/trafficcontrol/experimental/traffic_router_golang	[no test files]
?   	github.com/apache/trafficcontrol/experimental/traffic_router_golang/availableservers	[no test files]
?   	github.com/apache/trafficcontrol/experimental/traffic_router_golang/cgsrch	[no test files]
?   	github.com/apache/trafficcontrol/experimental/traffic_router_golang/config	[no test files]
?   	github.com/apache/trafficcontrol/experimental/traffic_router_golang/coveragezone	[no test files]
?   	github.com/apache/trafficcontrol/experimental/traffic_router_golang/crconfig	[no test files]
?   	github.com/apache/trafficcontrol/experimental/traffic_router_golang/crconfigdsservers	[no test files]
?   	github.com/apache/trafficcontrol/experimental/traffic_router_golang/crconfigpoller	[no test files]
?   	github.com/apache/trafficcontrol/experimental/traffic_router_golang/crconfigregex	[no test files]
?   	github.com/apache/trafficcontrol/experimental/traffic_router_golang/crstates	[no test files]
?   	github.com/apache/trafficcontrol/experimental/traffic_router_golang/crstatespoller	[no test files]
?   	github.com/apache/trafficcontrol/experimental/traffic_router_golang/fetch	[no test files]
?   	github.com/apache/trafficcontrol/experimental/traffic_router_golang/httpsrvr	[no test files]
?   	github.com/apache/trafficcontrol/experimental/traffic_router_golang/ipmap	[no test files]
?   	github.com/apache/trafficcontrol/experimental/traffic_router_golang/nextcache	[no test files]
ok  	github.com/apache/trafficcontrol/experimental/traffic_router_golang/quadtree	0.006s
?   	github.com/apache/trafficcontrol/experimental/traffic_router_golang/toutil	[no test files]
```

The full package names do not display for you when you run the tests?

Reply:
You are right, I ran this under directory which is not in GOPATH, so when running from go path, i got the same result as yours, 
```
$ go test ./...
?       github.com/apache/trafficcontrol/experimental/traffic_router_golang     [no test files]
?       github.com/apache/trafficcontrol/experimental/traffic_router_golang/availableservers    [no test files]
?       github.com/apache/trafficcontrol/experimental/traffic_router_golang/cgsrch      [no test files]
?       github.com/apache/trafficcontrol/experimental/traffic_router_golang/config      [no test files]
?       github.com/apache/trafficcontrol/experimental/traffic_router_golang/coveragezone        [no test files]
?       github.com/apache/trafficcontrol/experimental/traffic_router_golang/crconfig    [no test files]
?       github.com/apache/trafficcontrol/experimental/traffic_router_golang/crconfigdsservers   [no test files]
?       github.com/apache/trafficcontrol/experimental/traffic_router_golang/crconfigpoller      [no test files]
?       github.com/apache/trafficcontrol/experimental/traffic_router_golang/crconfigregex       [no test files]
?       github.com/apache/trafficcontrol/experimental/traffic_router_golang/crstates    [no test files]
?       github.com/apache/trafficcontrol/experimental/traffic_router_golang/crstatespoller      [no test files]
?       github.com/apache/trafficcontrol/experimental/traffic_router_golang/fetch       [no test files]
?       github.com/apache/trafficcontrol/experimental/traffic_router_golang/httpsrvr    [no test files]
?       github.com/apache/trafficcontrol/experimental/traffic_router_golang/ipmap       [no test files]
?       github.com/apache/trafficcontrol/experimental/traffic_router_golang/nextcache   [no test files]
ok      github.com/apache/trafficcontrol/experimental/traffic_router_golang/quadtree    1.190s
?       github.com/apache/trafficcontrol/experimental/traffic_router_golang/toutil      [no test files]
```

Reply:
I still get full package paths when running the tests from outside the GOPATH. Not sure why you don't get full package paths, but the README should have them.

Reply:
Oh, I missed 6d5783ceec. Looks good